1-exercises/B-while-loop/exercise.js

```diff
@@ -1,14 +1,22 @@
 /*
     while loops can be useful when you want to execute some code as long as some condition is true.    
 
-    Using a while loop, complete the function below so it logs (using console.log) the first n even numbers as a comma-seperated string.
+    Using a while loop, complete the function below so it logs (using console.log) the first n even numbers as a comma-separated string.
     The list of numbers should start with 0. n is being passed in as a parameter.
 */
 
 function evenNumbers(n) {
-    // TODO
+  if (n > 0) {
+    let j = "";
+    let i = 0;
+    while (i < n) {
+      j = j + "," + i * 2;
+      i++;
+    }
+    console.log(j.slice(1, j.length));
+  }
 }
```

Comment:
This is ok but I think there are cleaner ways of doing this without the slice, which we can talk about